### ISSUE TITLE
Drag copy stuck on after copying item to another Copyq tab

### ISSUE THREAD

**Issue by jonpatterns** [hluk/CopyQ#1692]
**Describe the bug**
After drag copying an item from the active tab to another tab, upon going back to the active tab drag/copy mode becomes stuck on (without a single mouse click) with last copied item shown by the pointer. Sometimes drag/copy mode can be exited by pressing escape, other times copyq has the drag/copy mode stuck on and can't be exited, making copyq unusable and frustrating.

EDIT: going to a tab that hasn't had any items copied from deselects drag mode - but going back to a copied from tab and positioning mouse over items leads to drag/copy mode being entered into immediately (without clicking).

**To Reproduce**

(Edited in light of first reply)

1. Drag/copy an item from the active copyq tab into an inactive copyq tab

2. click on another tab

3. Go back to the previous tab (that was copied from), position mouse over highlighted item

**Expected behavior**
Drag/copy mode shouldn't be entered unless an item clicked on and dragged.

**Screenshots**
![Copyq - drag stuck on](https://user-images.githubusercontent.com/525951/123543525-dcfa3600-d746-11eb-9f10-025077cba302.png)
Drag copied mode entered and stuck on by positioning mouse over highlighted item (without clicking)

**Version, OS and Environment**
CopyQ Clipboard Manager v4.1.0
Qt: 5.15.2
KNotifications: 5.81.0
Compiler: GCC
Arch: x86_64-little_endian-lp64
OS: Manjaro Linux

XFCE

**Comment by hluk** [comment]
This looks like an issue in window manager or desktop environment.

> **To Reproduce**
> 
>     1. Drag/copy an item from the active copyq tab into an inactive copyq tab
> 
>     2. Go back to the active tab, position mouse over highlighted item

How did you get into inactive tab? Drag'n'drop does not change tabs on my end.

**Comment by jonpatterns** [comment]
Drag'n'drop doesn't changes tabs for me either. So I guess I missed a step, see 1a below:

1. Drag/copy an item from the active copyq tab into an inactive copyq tab

1a. click on another tab

2. Go back to the previous tab (that was copied from), position mouse over highlighted item

**Comment by hluk** [comment]
I don't think I can do anything about this. It looks like an issue in Qt or window manager.

I'm observing similar behavior on Sway Wayland compositor - if I drop the item on window border the mouse button release event does not seem to be registered by CopyQ and hence the drag'n'drop operation is not completed.

**Comment by jonpatterns** [comment]
Okay, thanks. The drag and drop is handled per tab?

A work around is just to used keyboard shorts instead of dragging.

**Comment by hluk** [comment]
> The drag and drop is handled per tab?

Yup, specifically from the list widget, but changing it probably would not have any effect.

I tested drag'n'drop in Dolphin file manager and it also gets to some strange state after dropping item on window frame (in Sway compositor) - I'm assuming it's because it does not get the mouse button release event.

**Comment by bonehead542** [comment]
I have this problem when I drag an image from the image tab to the clipboard tab, then back again. What's annoying is dragging from the image tab to firefox or gimp, and having a thumbnail (same size as image tab preview) stuck to the mouse pointer. This problem began after installing manjaro with kde. The problem was not present using sabayon and copyq 4.1 built from a github clone. Clicking on the copyq main window to try and shake the artifact away can result in a duplicate file on the focused copyq tab, on the kde toolbar, or desktop folder. Pressing escape makes the artifact disappear, but it will reappear as soon as the mouse is moved. But pressing escape then holding control key while left mouse clicking the copyq item will return things to normal, as will restarting copyq. 
Also, the "Tab for Urls with Titile and Icon" extension didn't work properly. Copying an url gave a tabName not defined error.
All the above problems appeared on this Manjaro KDE install using the community repo, the aur version, the flathub version, and the github  code build. And they also appeared when each of the plugins I use were enabled or not.
I've seen the the (closed) links regarding the issue, but thought maybe some details might help. Still using copyq, and thanks for your work.

**Comment by hluk** [comment]
I believe this will be fixed in the next release (after 5.0.0). I'm assuming this was caused by incorrectly initiating multiple drag operations in the app (#1809).

If you can, please test [the development flatpak build](https://github.com/flathub/com.github.hluk.copyq/pull/32), but it's safer to backup the app's configuration and data before testing.

```bash
# Close running instance of the app
flatpak run com.github.hluk.copyq exit

# (Optional) Backup the config directory
dir=$(dirname "$(flatpak run com.github.hluk.copyq info config)")
cp -rv "$dir" ~/Documents/copyq-backup-"$(date -Iseconds)"

# Install the dev version
flatpak install https://dl.flathub.org/build-repo/65451/com.github.hluk.copyq.flatpakref

flatpak run com.github.hluk.copyq
```

**Comment by DUOLabs333** [comment]
I'm building it now from git, I'll see what happens.
